packages/block-library/src/search/index.php

```diff
@@ -29,11 +29,16 @@ function render_block_core_search( $attributes ) {
 		)
 	);
 
-	$input_id            = wp_unique_id( 'wp-block-search__input-' );
-	$classnames          = classnames_for_block_core_search( $attributes );
-	$show_label          = ! empty( $attributes['showLabel'] );
-	$use_icon_button     = ! empty( $attributes['buttonUseIcon'] );
-	$show_button         = ( ! empty( $attributes['buttonPosition'] ) && 'no-button' === $attributes['buttonPosition'] ) ? false : true;
+	$input_id        = wp_unique_id( 'wp-block-search__input-' );
+	$classnames      = classnames_for_block_core_search( $attributes );
+	$show_label      = ( ! empty( $attributes['showLabel'] ) ) ? true : false;
+	$use_icon_button = ( ! empty( $attributes['buttonUseIcon'] ) ) ? true : false;
```

Comment:
[`empty`](https://www.php.net/empty) returns a `bool` value. In effect, the ternary operator is redundant here.

Reply:
Good catch, thank you. 

Looks like I messed up the merge with trunk because this was changed in https://github.com/WordPress/gutenberg/pull/66857

Reply:
Fixed it in bdaaf2e3eaf604fd4747eaf5bf49c5d6d03d7a94